### DISCUSSION TITLE
Newbie question: do Minio always store 4 copies of files?

### DISCUSSION THREAD

**Discussion by meotimdihia** [minio/minio#13198] [Q&A] [answered]
Hi, I start a Minio server with this command: 
```shell
minio server --console-address ":9001"  /mnt/disk{1...4}
```
```shell
storage_class standard=EC:2 rrs=EC:2
```
I tried to upload 1 file and  Minio always creates 4 copies. Do we set up it to 2 copies only?

**Comment by klauspost** [comment, marked as answer]
@meotimdihia It doesn't. In your setup the file will be split into 2 data shards and 2 erasure shards. For for this setup you have 2x the data and any 2 of your disks can go bad and you can still recover the data.

See https://docs.min.io/minio/baremetal/concepts/erasure-coding.html

**Reply by meotimdihia** [reply]
Thanks for the explanation. I didn't think the file is a shard. 😁
